modules/mbedtls/Kconfig.tls-generic

```
@@ -567,6 +568,37 @@ config MBEDTLS_PSA_P256M_DRIVER_RAW
 	  Warning: Usage of this Kconfig option is prohibited in Zephyr's codebase.
 	           Users can enable it in case of very memory-constrained devices, but be aware that the p256-m interface is absolutely not guaranted to remain stable over time.
 
+config MBEDTLS_PSA_STATIC_KEY_SLOTS
+	bool "Use statically allocated key buffers to store key material"
+	default y if !MBEDTLS_ENABLE_HEAP
+	help
+	  By default Mbed TLS's PSA Crypto core uses heap memory to store the
+	  key's material for each key slot. This might impose an undesired
+	  requirement to support heap memory and its management code affecting
+	  RAM and ROM footprints at the same time.
+	  Enabling this symbol cause Mbed TLS to pre-allocate all the key slot
+	  buffers that are used to store the key's material at built time, thus
+	  removing the need for heap memory. Each buffer will be sized to
+	  contain the largest asymmetric/symmetric key type enabled in the build
+	  through PSA_WANT symbols.
+
+config MBEDTLS_PSA_KEY_SLOT_COUNT
+	int "Number of key slots in PSA Crypto core"
+	default 16
+	help
+	  Set the number of key slots that are available in the PSA Crypto core.
+	  Be aware that each slots, even if unused, increases RAM memory
```

```suggestion
	  Be aware that each slots, even if unused, increases RAM
```
Random Access Memory memory :smiley: